lms/djangoapps/lti_provider/management/commands/rescore_lti.py

```diff
@@ -0,0 +1,45 @@
+"""
+Management command to rescore all lti problems for the given course for all users.
+"""
+
+import textwrap
+
+from django.core.management import BaseCommand
+
+from opaque_keys.edx.keys import CourseKey
+
+from lti_provider.models import GradedAssignment
+from lti_provider import tasks
+from student.models import CourseEnrollment
+
+
+class Command(BaseCommand):
+    """
+    Rescore all lti problems for the given courses for all users.
+
+    Examples:
+
+        ./manage.py rescore_lti course-v1:edX+DemoX+Demo_Course course-v1:UBCx+course+2016-01
```

Comment:
The `lti_provider` app is added conditionally in aws.py.  Try on jcdyer.sandbox.edx.org.